pkg/scanners/clairv4/clairv4.go

```diff
@@ -0,0 +1,122 @@
+package clairv4
+
+import (
+	"crypto/tls"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/httputil/proxy"
+	"github.com/stackrox/rox/pkg/registries"
+	"github.com/stackrox/rox/pkg/scanners/types"
+	"github.com/stackrox/rox/pkg/urlfmt"
+)
+
+const (
+	requestTimeout = 2 * time.Minute
```

Comment:
Seems high for a timeout.

Reply:
This was required when I tested it locally [here](https://github.com/stackrox/stackrox/pull/3768). Going to keep it for now, but we can always update if needed